### PR TITLE
Add committee the ability to vote for new inner ring list

### PR DESCRIPTION
**Pull request by alexvanin**
Now committee can change inner ring list by collecting committee votes. However new inner ring list proposed by committee should fit the same criteria: it should contain keys from candidate list and at most 1\3 of new nodes.

### CODE REVIEW

**Comment by alexvanin** [comment]
Checked on a single node environment with stand by committee.

File: neofs/neofs_contract.go [hunk 1]
```diff
@@ -520,6 +531,17 @@ func innerRingInvoker(ir []node) []byte {
 	return nil
 }
 
+func committeeInvoker(cmt [][]byte) []byte {
+	for i := 0; i < len(cmt); i++ {
+		node := cmt[i]
+		if runtime.CheckWitness(node) {
```
Comment (roman-khimov) [inline neofs/neofs_contract.go:537]: This means that each committee member has to invoke `innerRingUpdate` and I'm not sure this is the way it's supposed to work. At the same time you probably can't really check for committee multisignature.

File: neofs/neofs_contract.go [hunk 2]
```diff
@@ -338,9 +343,15 @@ func InnerRingUpdate(chequeID []byte, args [][]byte) bool {
...
 
-	irKey := innerRingInvoker(irList)
-	if len(irKey) == 0 {
-		panic("innerRingUpdate: invoked by non inner ring node")
+	voteKey := innerRingInvoker(irList)
+	if len(voteKey) == 0 {
+		committee := getCommitteeKeys()
+		threshold = len(committee)/3*2 + 1
+
+		voteKey = committeeInvoker(committee)
```
Comment (roman-khimov) [inline neofs/neofs_contract.go:351]: And this treats IR votes and committee votes equal, I'm not sure about that either.

**Review by roman-khimov — COMMENTED** [review]
I think the intention for committee-based voting was to use Designate contract and add a role there for IR nodes. Although moving IR nodes list there will also require some kind of synchronization between two contracts and Designate generates no events at the moment.

**Comment by alexvanin** [comment]
I am moving this PR to draft, since there will be changes in new committee/innerring communication scheme. It will require support of committee signagure check in contract.

**Comment by alexvanin** [comment]
I close this PR because there will be updated NeoFS governance scheme that has nothing to do with this PR (#32)

**Pull request closed without merging**